docs/releases/v1.4.2.md

```diff
@@ -0,0 +1,30 @@
+# Release notes
+
+
+## Changelog
+
+Changes between `1.4.0` and this release: `1.4.1`
+
+- Refactor `dual-nginx` package to use the base of the `nginx` *(simple)* deployment.
```

Comment:
There's no mention of the change to Kustomize v3.

Reply:
About kustomize v3, it was already running using kustomize v3. This PR uses kustomize v3 features to avoid code duplication :)

Reply:
Mmm, correct me if I'm wrong but the code was kustomize 3 compatible, it wasn't ported to Kustomize 3 before. In fact the readme on master says `Kustomize >= v1`, but I see it's been changed now, so I'm marking this as resolved.